actions/check-commit-messages/README.md

```diff
@@ -0,0 +1,40 @@
+# Check commit messages
+
+This action checks the commit messages on a branch to ensure they follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification. It assumes [Python](https://www.python.org/), [pip](https://pip.pypa.io/en/stable/) and [Commitizen](https://commitizen-tools.github.io/commitizen/) are already setup, see the [setup Python action](../setup-python/README.md) and [setup Commitizen action](../setup-commitizen/README.md) for ready to use actions to do this.
+
+## Action inputs
+
+| Input             | Description                                                             | Default                                           |
+| ----------------- | ----------------------------------------------------------------------- | ------------------------------------------------- |
+| `starting_commit` | The commit to start checking from. By default, all commits are checked. | `'${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}'` |
+
+## Example usage
+
+```yaml
+name: 'Check commit messages'
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  check_commit_messages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout repository'
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: 'Setup Python'
+        uses: dfinity/actions/actions/setup-python@main
+
+      - name: 'Setup Commitizen'
+        uses: dfinity/actions/actions/setup-commitizen@main
```

Comment:
are these 2 actions a requirement for the `check-commit-messages`?

Reply:
just saw now in the description that it is, maybe would be worth to have the action run both if not available instead of making the user be aware of it

Reply:
I've checked what will happen on this branch: https://github.com/dfinity/actions/pull/8
Calling these actions twice won't fail the pipeline and the subsequent runs are executed really quickly due to caching.